docs/assets/diagrams/linking/3-unlinking.puml

```diff
@@ -0,0 +1,58 @@
+@startuml Unlinking
+
+title: Unlinking
+hide footbox
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  participant "Firebase" as Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+'start flow
+Alice -> App++: I want to unlink this bank account
+
+App -> Server ++: ""**DELETE /consents/123**""
+App <-- Server: ""202 Accepted""
+deactivate App
+
+Server -> Adapter ++: ""**DELETE /consents/123**""
+Adapter <-- Server: ""202 Accepted""
+deactivate Server
+
+Adapter -> Server++: ""**PATCH /consents/123** \n{\n status: string,\n revokedAt: string (of timestamp)\n}""
+Adapter <-- Server: ""200 OK""
+deactivate Adapter
+
+rnote left of Server #Light
+  ""firebase.firestore()""
+  ""  .collection("consents")""
+  ""  .where("consentId", "==", "123")""
+  ""  .set({status: "REVOKED", revokedAt: "timestamp"})""
+end rnote
+
+Server -> Firebase: The account has been unlinked.
+Server <-- Firebase: ""200 OK""
+deactivate Server
+
+rnote left of Firebase #Light
+  ""event(""
+  ""  "document.set",""
+  ""  "consents/abc123"""
+  "" ({status: "REVOKED", revokedAt: "timestamp"})""
```

Comment:
```suggestion
  "" <snapshot object>""
```

Reply:
dont we want to be clear what's passed back

Reply:
Just to avoid code duplication since it just "forwards" all the changes that the server made to the firebase doc

Reply:
ah i see
i think both ways are fine (just a style thing) - I'd prefer to stick with this as it follows PISP linking diagram format. But since you own the demo server upto you - happy with either format

Reply:
I think it would be better to conform with the other styling for consistency, we can always change them in the future :)

The main issue is because for other sequence diagrams the snapshot object is quite big and it would consume a lot of space for such repetition.

Reply:
okay makes sense - I'll make the changes rn - and if you could go ahead and approve this I'll merge it in